content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,160 @@
+
+== Documentation Update
+
+Contributed by: https://github.com/kmartens27[Kevin Martens]
+
+This year, we saw documentation contributions from new and seasoned Jenkins users.
+These contributions included blog posts, documentation additions and updates, documentation migration, and other improvements.
+All of this combined has helped expand and empower the Jenkins community.
+
+The Jenkins project saw 48 blog posts, submitted by 23 different authors, and overall 814 PRs were merged.
+The contributions are a result of the community and collaboration, with various projects throughout the year such as She Code Africa, Google Summer of Code, and Hacktoberfest.
+
+=== Pipeline Steps Reference
+
+Thanks to the work of https://www.jenkins.io/blog/authors/vihaanthora/[Vihaan Thora], contributing via https://www.jenkins.io/blog/2022/10/10/pipeline-steps-improvement-gsoc-report/#project-specific-guidance[Google Summer of Code], the https://www.jenkins.io/doc/pipeline/steps/[Pipeline steps] documentation has been improved.
+The Pipeline Steps reference guide is invaluable for developers when working in Jenkins and utilizing plugins.
+The updates include search functionality, overall appearance, and a reduction in page load.
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/image5.png[image,width=624,height=388]
+
+=== Algolia search
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/image6.png[image,width=275,height=52]
+
+The Jenkins site search has been updated to use the latest version of https://www.algolia.com/[Algolia].
+We want to recognize and thank https://www.jenkins.io/blog/authors/halkeye/[Gavin Mogan] for all of his work on this and Algolia for donating the search functionality.
+The site search now provides more relevant results and suggestions for users.
+A visual update was included as part of the upgrade, resulting in the new look and UI.
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/image7.png[image,width=363,height=317]
+
+== Advocacy & Outreach Update
+
+Contributed by: https://github.com/alyssat[Alyssa Tong]
+
+* Jenkins in GSoC 2023 : https://www.jenkins.io/projects/gsoc/2023/project-ideas/[Call for Project Ideas] + https://www.jenkins.io/blog/2022/12/09/GSoC-the-gift-of-mentorship/[Call for Mentors].
+** Watch https://youtu.be/k_sTkGtTix8[A Guide to Better Preparations] for potential GSoC candidates who want to get started and increase their chance of getting accepted into the program.
+* https://fosdem.org/2023/[FOSDEM’23]: Jenkins will have a devstand at FOSDEM (Feb 4-5, 2023).
+* https://www.socallinuxexpo.org/scale/20x[SCALE 20X]: Jenkins will have a booth presence at SCALE (March 9-12, 2023).
```

Comment:
```suggestion
* https://www.socallinuxexpo.org/scale/20x[SCALE 20X]: Jenkins will have a booth presence at SCALE (March 9-12, 2023).
* New release leads
** Newer members and contributors to Jenkins are taking on the role of release lead for our LTS releases.
This allows people the opportunity to work directly with Jenkins developers and maintainers on the ever evolving platform.
It also provides the community another avenue to make their voices heard and influence Jenkins development.
```
adding new release leads blurb under advocacy and outreach as this is an expansion of the community